tests/ast/test_basic_evaluation.py

```diff
@@ -71,6 +74,24 @@ def test_J2(self):
         self.sp.set_value('Sheet1!B2', 10)
         self.assertEqual(self.sp.evaluate('Sheet1!J2'), 0)
 
+    def test_J22(self):
+        cell = self.sp.cellmap['Sheet1!J22']
+        assert cell.value == 'Paul'
+
+        assert self.sp.evaluate('Sheet1!J22') == 'Paul'
```

Comment:
Noticed I have inconsistently used `assert` (for pytest) and `assertEqual` (legacy unittest standard). Is it OK if I change my `assertEqual` to `assert`?

Reply:
Going to switch my additions to asserts, because they work far better with pytest which is used by travis and tox.